src/main/java/ti4/image/MapGenerator.java

```diff
@@ -633,8 +633,7 @@ private int drawPriorityTrack(int y) {
             g2.drawRect(i * boxWidth + trackXShift, y, boxWidth, boxHeight);
         }
 
-        int colCount = priorityTrack.size() - 1;
```

Comment:
bug fix: When printing OmegaPhase's Priority Track, `colCount` was a holdover from the VP track code I copied. It doesn't apply here (always 1 column), and was causing tokens to print off-center when player count was >2.